content/en/monitors/service_level_objectives/monitor.md

```diff
@@ -51,6 +51,14 @@ an SLO breach.
 Here we add contextual information about the purpose of the SLO, including any related information
 in the description and tags you would like to associate with the SLO.
 
+#### Underlying monitor and SLO histories
+
+Making changes to the monitor used by an SLO recalculates the SLO history.  Therefore, the monitor history and SLO history may not match after a monitor update.
+
+SLO calculations do not take into account when a monitor is resolved manually or as a result of the **_After x hours automatically resolve this monitor from a triggered state_** setting.  If these are important tools for your workflow, consider cloning your monitor, removing auto-resolve settings and `@-notification`s, and using the clone for your SLO.
```

Comment:
Or should we suggest using metric-based SLOs?

Reply:
We shouldn't suggest it because it depends on the use case but we can remind them to determine if they are using the right one. 

Confirm you are using the preferred SLI type for your use case. In Datadog we support monitor-based SLIs as described here or you can also use metric-based SLIs as described here(https://docs.datadoghq.com/monitors/service_level_objectives/event/)

Reply:
Added @meghanelizabeth 's suggestion below